staging/src/k8s.io/client-go/transport/cache_go118.go

```diff
@@ -0,0 +1,22 @@
+//go:build go1.18
```

Comment:
Made this a new file with a build constraint in case someone was using an older go version.

Reply:
should we force the go version too?